sqlx_user_crud/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "sqlx_user_crud"
+version = "0.1.0"
+authors = ["james.meyer"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix-web = "3"
+actix-rt = "1.1.1"
+sqlx = { version = "0.3.5", features = ["mysql"] }
```

Comment:
sqlx is at 0.4.2

Reply:
Sqlx was probably on 0.3.5 when I started or I encountered a bug and had to roll down in versions. I've attempted to upgrade and it looks like they broke compatibility with the `execute` method, which now returns some type called a `MySqlDone`. Also some lifetimes for the `FromRow` trait seem to have been changed. Moving to 0.4.2 will introduce a lot of refactoring. Is it a requirement that this PR be on the most up to date version of each package?

Reply:
The done types all implement `sqlx::Done`, which provides a `.rows_affected()` method to obtain what was previously just the return value of the execute method. Re. `FromRow` you can usually derive it.

Reply:
it's fine, but document the reasons (in `Cargo.toml`) and what requirement there are for an upgrade to work

Reply:
Due to some limitations with the pipeline, I moved the source code of this project to an external repository, but I still implemented your suggestion in the external repo.